src/js/control-bar/progress-control/mouse-time-display.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file mouse-time-display.js
+ */
+import Component from '../../component.js';
+import * as Dom from '../../utils/dom.js';
+import * as Fn from '../../utils/fn.js';
+import formatTime from '../../utils/format-time.js';
+import throttle from 'lodash-compat/function/throttle';
+
+/**
+ * The Mouse Time Display component shows the time you will seek to
+ * when hovering over the progress bar
+ *
+ * @param {Player|Object} player
+ * @param {Object=} options
+ * @extends Component
+ * @class MouseTimeDisplay
+ */
+class MouseTimeDisplay extends Component {
+
+  constructor(player, options) {
+    super(player, options);
+
+    this.position = 0;
+    this.newTime = 0;
+    this.update();
+
+    player.on('ready', () => {
+      this.on(player.controlBar.progressControl.el(), 'mousemove', throttle(Fn.bind(this, this.handleMouseMove), 50));
```

Comment:
It feels a little jumpy to me at 20fps. It's nit-picky but I think we could bump it up to at least 33ms without impacting performance.